stress/shell/src/main/java/alluxio/stress/cli/UfsIOBench.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.conf.InstancedConfiguration;
+import alluxio.job.plan.PlanConfig;
+import alluxio.stress.BaseParameters;
+import alluxio.stress.job.IOConfig;
+import alluxio.stress.worker.IOTaskResult;
+import alluxio.stress.worker.WorkerBenchParameters;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.underfs.UnderFileSystemConfiguration;
+import alluxio.util.CommonUtils;
+import alluxio.util.executor.ExecutorServiceFactories;
+
+import com.beust.jcommander.ParametersDelegate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+/**
+ * A benchmark tool measuring the IO to UFS.
+ * */
+public class UfsIOBench extends Benchmark<IOTaskResult> {
+  private static final Logger LOG = LoggerFactory.getLogger(UfsIOBench.class);
+  private static final int BUFFER_SIZE = 1024 * 1024;
+
+  @ParametersDelegate
+  private WorkerBenchParameters mParameters = new WorkerBenchParameters();
+
+  private final InstancedConfiguration mConf = InstancedConfiguration.defaults();
+  private final HashMap<String, String> mHdfsConf = new HashMap<>();
+
+  @Override
+  public PlanConfig generateJobConfig(String[] args) {
+    // remove the cluster flag
+    List<String> commandArgs =
+            Arrays.stream(args).filter((s) -> !BaseParameters.CLUSTER_FLAG.equals(s))
+                    .filter((s) -> !s.isEmpty()).collect(Collectors.toList());
+
+    commandArgs.addAll(mBaseParameters.mJavaOpts);
+    String className = this.getClass().getCanonicalName();
+    return new IOConfig(className, commandArgs, mParameters);
```

Comment:
this test couldn't re-use the existing definition? It someone defeats the purpose with only using a small portion of the existing stress benchmark?